specs/plugin-extensibility.md

```diff
@@ -479,13 +479,20 @@ This allows Notary v2 implementations to perform the same logic themselves, if t
     "criticalAttributes" : 
     { 
        "contentType" : "application/vnd.cncf.notary.payload.v1+json",
-       // One of notary.default.x509 or notary.signingAuthority.x509
-       "signingScheme" : "notary.default.x509",
+       // One of notary.x509 or notary.signingAuthority.x509
+       "signingScheme" : "notary.x509" | "notary.signingAuthority.x509",
        // Value is always RFC 3339 formatted date time string
-       "expiry": "2022-10-06T07:01:20Z", 
+       "expiry": "2022-10-06T07:01:20Z",
+       // Populate authenticSigningTime critical attibute 
+       // if signingScheme is notary.signingAuthority.x509.
+       "authenticSigningTime": "2022-04-06T07:01:20Z",
```

Comment:
Should we reword this to say
`authenticSigningTime must be present, if signingScheme is notary.signingAuthority.x509.`